### DISCUSSION TITLE
Is this compatible with DaisyUI colors?

### DISCUSSION THREAD

**Discussion by marshallswain** [windicss/windicss#194] [Q&A] [answered]
I'm attempting to get DaisyUI to work with Windi.

I'm specifically trying to get the colors to work:
https://daisyui.netlify.app/docs/install
![Screen Shot 2021-03-25 at 8 59 31 PM](https://user-images.githubusercontent.com/128857/112571128-02c11900-8dad-11eb-8ea2-ccf57549fb25.jpg)

This page shows the classes I would expect to see:
https://daisyui.netlify.app/core/colors/

The source code for their colors is found here: https://github.com/saadeghi/daisyui/blob/master/colors.js#L34
Each color is a function that receives `(opacityVariable, opacityValue)`.  I skimmed through the windi source code, but wasn't able to determine at a glance if this would work.  I would really appreciate it if you could let me know if this should work.  Thank you!

**Comment by marshallswain** [comment]
Oops.  I meant for this to go in the windicss repo.  I got my tabs mixed up.

**Reply by antfu** [reply]
Np, tranferred.

**Comment by marshallswain** [comment]
I've made a little progress in getting some of the styles to work by using the `transform` utility from `windicss/helpers`

```js
const { transform } = require('windicss/helpers');

module.exports = {
  plugins: [transform('daisyui')],
  theme: {
    extend: {
      colors: require('daisyui/colors')
    }
  }
}
```

Still figuring out the colors.

**Comment by marshallswain** [comment]
I've created a PR that fixes this issue: https://github.com/windicss/windicss/pull/198

**Comment by marshallswain** [comment, marked as answer]
Question: Does with work with DaisyUI?

Answer: [PR 1](https://github.com/saadeghi/daisyui-demo/pull/1), [PR 2](https://github.com/saadeghi/daisyui/pull/16), [PR 3](https://github.com/windicss/windicss/pull/198). Yes.

![Screen Shot 2021-03-27 at 10 22 39 AM](https://user-images.githubusercontent.com/128857/112727091-61d67900-8ee6-11eb-913f-49b565cf3719.jpg)
